### ISSUE TITLE
PermissionError: [WinError 32] 另一个程序正在使用此文件，进程无法访问。

### ISSUE THREAD

**Issue by stephenwang0317** [hulianyuyy/STGAT#9]
Python version:3.9
PyTorch version: 1.12.1+cuda113
sysinfo: win10 22H2
When I run train.py according to readme, the program first print the infomation of python version and system and Good Luck. Then it print config infomation on the console. However, the program print the information of python version and Good Luck again with an Error shown in the title.

**Comment by hulianyuyy** [comment]
I haven't encountered this problem and haven't run it on the windows system. But according to others' reply, this code should work well on both systems. The printed information is correct. I suspect that it seems that  the code  is conducted twice but i can't figure it out. You can check it again or provide me  with more information.

**Comment by stephenwang0317** [comment]
I debug the code and found that in the function train_classifier of train_val_model.py on line 48, 
`process = tqdm(IteratorTimer(data_loader), desc='Train: ', dynamic_ncols=True)`
when the IteratorTimer initiate, the error occured.
I searched on the Internet and found a solution is that add `if __name__ == '__main__'`  at the beginning of the code.
I added it in train.py before line 48 which is `with TimerBlock("Good Luck") as block:`  but it didn't work.
Finally I set the parameter _worker_  in the config file to 1 and it worked.
